config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,6 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
+        #image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
why this comment?

Reply:
Image is updated to v0.8.0 in the latest commit